language/stdlib/modules/transaction_fee.move

```diff
@@ -80,11 +78,10 @@ module TransactionFee {
 
             LibraAccount::pay_from_capability<Token>(
                 addr,
-                Vector::empty(),
+                x"",
                 &distribution_resource.fee_withdrawal_capability,
                 amount_to_distribute_per_validator,
-                // FIXME: Update this once we have bytearray literals
-                AddressUtil::address_to_bytes(0xFEE),
+                x"",
```

Comment:
Did you intend to replace that 0xfee with an empty vector?

Reply:
Good catch; didn't mean to change the semantics here.

However, I  think this probably ok. This will get packaged into metadata that will be emitted as part of the `ReceivedPaymentEvent` for validator payments. I don't think this particular metadata is useful, and it makes these events (which happen every block) slightly bigger, so probably worth killing? CC @tzakian to double-check